app/src/main/java/pl/droidevs/books/savebook/SaveBookViewModel.java

```diff
@@ -75,18 +87,18 @@ public void onSubscribe(Disposable d) {}
 
                     @Override
                     public void onComplete() {
-                        successWithAdding.postValue(true);
+                        successWithSaving.postValue(true);
```

Comment:
Wyraźnie brakuje u nas takiej klasy: https://github.com/googlesamples/android-architecture-components/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/vo/Resource.java

Przerefaktorze to później 😉